pypeit/spectrographs/p200_ngps.py

```diff
@@ -0,0 +1,619 @@
+            :class:`~pypeit.par.pypeitpar.PypeItPar`: Parameters required by
+            all of PypeIt methods.
+        """
+        par = super().default_pypeit_par()
+
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+        par['calibrations']['slitedges']['edge_thresh'] = 10. # Lower edge tracing thresdhold to catch leftmost slit
+
+        par['calibrations']['slitedges']['minimum_slit_length'] = 100 # Set minimum slit length 
+
+        par['calibrations']['slitedges']['add_slits'] = ['1:2090:24:153']
+        
+        par['scienceframe']['process']['combine'] = 'median'
+        par['calibrations']['standardframe']['process']['combine'] = 'median'
+
+        par['scienceframe']['process']['use_overscan'] = True
+        par['scienceframe']['process']['sigclip'] = 4.0
+        par['scienceframe']['process']['objlim'] = 5.0
+
+        # Make a bad pixel mask
+        par['calibrations']['bpm_usebias'] = True
+
+        # Set pixel flat combination method
+        par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
+
+        par['calibrations']['wavelengths']['lamps'] = ['ThAr'] # FeAr and ThAR lamps for NGPS (ThAr)
+        par['calibrations']['wavelengths']['method'] = 'full_template' # Use wavelength template
+        par['calibrations']['wavelengths']['reid_arxiv'] = 'wvarxiv_p200_ngps_20250131T1227.fits' # R Channel
+
+        par['calibrations']['wavelengths']['rms_thresh_frac_fwhm'] = 1.0 
+
+        # Do not flux calibrate
+        #par['fluxcalib'] = None
+        par['sensfunc']['algorithm'] = 'UVIS' # CHANGE #
+
+        # Set the default exposure time ranges for the frame typing
+        par['calibrations']['biasframe']['exprng'] = [None, 0.001]
+        par['calibrations']['darkframe']['exprng'] = [999999, None]     # No dark frames
+        par['calibrations']['pinholeframe']['exprng'] = [999999, None]  # No pinhole frames
+        par['calibrations']['arcframe']['exprng'] = [None, 120]
+        par['calibrations']['standardframe']['exprng'] = [None, 120]
+        par['scienceframe']['exprng'] = [90, None]
+
+        return par
+
+
+    def config_specific_par(self, scifile, inp_par=None): 
+        """
+        Modify the PypeIt parameters to hard-wired values used for
+        specific instrument configurations.
+
+        Args:
+            scifile (:obj:`str`):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`~pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`~pypeit.par.parset.ParSet`: The PypeIt parameter set
+            adjusted for configuration specific parameter values.
+        """
+        # Start with instrument wide
+        par = super().config_specific_par(scifile, inp_par=inp_par)
+
+        return par
```

Comment:
If you're always using the base class function, you can remove it from this class.